src/client/src/rt-theme/index.ts

```diff
@@ -12,8 +11,11 @@ export { themes, getThemeColor } from './themes'
 export type { Theme, TouchableIntentName, ColorProps, ThemeSelector } from './themes'
 export { colors } from './colors'
 export { default as GlobalStyle } from './globals'
-
-export const styled: ThemedStyledInterface<Theme> = baseStyled
-
 export { ThemeName, ThemeProvider, ThemeConsumer, useTheme } from './ThemeContext'
 export { default as ThemeStorageSwitch } from './ThemeStorageSwitch'
+
+// Make all styled-component functions (e.g. `styled`, `css`) typed with Theme
```

Comment:
This is the current recommended way to get typed interpolations in styled components, rather than the proxy approach we previously had